Update New-CsTeamsMeetingPolicy.md (#9131)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md

Accept wildcard characters: False
```

### -AllowNetworkConfigurationSettingsLookup

> Determines whether network configuration setting lookup can be made for users who are not enterprise voice enabled, it will be used for enabling network roaming policy

Reviewer: ```suggestion
Determines whether network configuration setting lookups can be made by users who are not Enterprise Voice enabled. It is used to enable Network Roaming policies.
```
